packages/rainbowkit/src/components/ConnectOptions/DesktopOptions.tsx

```diff
@@ -174,61 +209,149 @@ export function DesktopOptions({ onClose }: { onClose: () => void }) {
           wallet={selectedWallet}
         />
       );
-      headerLabel = hasQrCode && `Get started with ${selectedWallet.name}`;
+      headerLabel =
+        hasQrCode &&
+        `Get started with ${selectedWallet.shortName || selectedWallet.name}`;
```

Comment:
Just confirming, we want to use the short name even when we're not in compact mode?

Reply:
yeah - this really is just for coinbase, "coinbase wallet" felt crowded in this screen even on default mode. i can make it the long string on default to keep consistent as much as possible 👍